src/main/java/frc/robot/Robot.java

```diff
@@ -187,7 +189,13 @@ public void robotInit() {
     elevator.setDefaultCommand(elevator.setExtensionCmd(() -> 0.0));
     feeder.setDefaultCommand(
         Commands.repeatingSequence(
-            feeder.indexCmd().until(() -> currentTarget != Target.SPEAKER),
+            feeder
+                .indexCmd()
+                .until(
+                    () ->
+                        currentTarget != Target.SPEAKER
+                            || currentTarget != Target.FEED
+                            || currentTarget != Target.SUBWOOFER),
```

Comment:
Having a little trouble following this. What's the idea here?

Reply:
the feeder should run indexing until the target is changed to be something that isnt one of the speaker modes. right now thats just the amp, and it could be refactored to reflect that. it was more readable before this refactor, see if the refactor around jacobs comment makes it better

Reply:
So should this be && instead of ||? Right now this always evaluates to `true`, doesn't it?